### ISSUE TITLE
Table 'cachet.metrics' doesn't exist

### ISSUE THREAD

**Issue by dknx01** [CachetHQ/Cachet#2525]
Hi,
I'm using Cachet 2.3.9 and have the following error while reading metric points:
Table 'cachet.metrics' doesn't exist

I use a table prefix, as requested during install. The prefix is not used when selecting the metric data for the charts.
I think it is related to #1938.

The problem is inside the _app/Repositories/Metric/MySqlRepository.php_ file.

**Comment by ConnorVG** [comment]
In your `config/database.php` file, under the `default` key, what is the value of the `prefix` key?

**Comment by dknx01** [comment]
the prefix is "lbc"

**Comment by ConnorVG** [comment]
What happens after you run:
```bash
php artisan cache:clear
php artisan route:cache
php artisan config:cache
```

**Comment by jbrooksuk** [comment]
Did I not fix this on Cachet 2.3?
